server/build.sbt

```diff
@@ -67,7 +67,7 @@ lazy val root = (project in file("."))
       // Autovalue
       "com.google.auto.value" % "auto-value-annotations" % "1.9",
       "com.google.auto.value" % "auto-value" % "1.9",
-      "com.google.auto.value" % "auto-value-parent" % "1.9",
+      "com.google.auto.value" % "auto-value-parent" % "1.9" pomOnly(),
```

Comment:
And alternate approach would be to use the script to modify this line in the generated pom

Reply:
Agreed that we could modify the generated pom. However, I'd prefer to do as little hand-editing as we can. The `auto-value-parent` package doesn't ever have a JAR, so this change should be a no-op. That said, good call on manually testing.

I confirmed that `bin/run-dev` continues to load. Additionally, I ran the docker build from the [build and push script](https://github.com/seattle-uat/civiform/blob/cc9628a5f398c5d19720f8acdb06bdadf90ab036/bin/build-and-push#L5-L17). I didn't think it necessary to push an artifact to DockerHub since the build itself is what we're verifying didn't change. Finally, passing browser / unit test should hopefully bolster confidence. Updated the "Tested" section of the PR to reflect this.

Reply:
Awesome!